### ISSUE TITLE
Inventory-next production environment ready

### ISSUE THREAD

**Issue by FuhuXia** [GSA/data.gov#2272]
## User Story

Now that inventory-next has been tested in the staging environment, complete the following set up steps to get inventory-next running on production for dry-runs and final switchover.

## Details/tasks
- [x] [Request DNS](https://github.com/GSA/datagov-deploy/wiki/DNS) for inventory-next.data.gov (ticket to BSP)
- [x] [Request SSL certificate](https://github.com/GSA/datagov-deploy/wiki/TLS-SSL-Certificates) for inventory-next.data.gov (ticket to BSP)
- [x] New Load Balancer and/or Netscaler configuration in BSP (ticket to BSP)
- [x] New stand-alone redis instances ~~provisioned with TLS encryption~~( ticket to BSP)
- [x] New Solr instances #2284
- [x] New CKAN DB and Datastore DB loaded with recent production snapshots
- [x] Deploy inventory-next in production #2285

## Acceptance Criteria

- [x] GIVEN inventory-next has been deployed to production \
  AND I spoofed set up has been competed \
  AND set up has been competed \
  WHEN I browse https://inventory-next.data.gov \
  THEN I see the inventory home page \
  OR browse request causes CKAN error and new ticket is created to resolve.

## Security Considerations ([required](https://nvd.nist.gov/800-53/Rev4/control/CM-4))
[comment]: # "Our SSP says 'The Data.gov team ensures security implications are considered as part of the agile requirements refinement process by including a section in the issue template used as a basis for new work.' so please don't remove this section without care."
The deployment will use the usual processes for staging/production rollouts, and not introduce any new process or components where security problems are likely to appear.

**Comment by FuhuXia** [comment]
RITM0778598 was created for DNS
RITM0778768 was created for SSL certificate
RITM0778842 was created for NetScaler configuration

**Comment by FuhuXia** [comment]
RITM0778598 DNS ticket was rejected due to improper request. DNS wiki updated.
RITM0778768 SSL certificate ticket: We were given an unchanged cert file. They are working in it to correct it.
RITM0778842 NetScaler ticket was completed.

**Comment by FuhuXia** [comment]
DNS: new ticket RITM0782441 was created.

SSL certificate: the new cert is generated and verified. RITM0782462 is created to request install the the new cert on NetScaler.

**Comment by FuhuXia** [comment]
✅  DNS: RITM0782441 is complete. inventory-next.data.gov is ready.

🚧  SSL certificate: RITM0782462 ticket is still being worked on to install SSL certificate in BSP Netscaler and PA. There were some recent firewall decryption changes making things complicated.

**Comment by FuhuXia** [comment]
🚧  SSL certificate: RITM0782462 still pending.
🚧  RDS renaming: RITM0783916 created to rename RDS endpoints.

**Comment by FuhuXia** [comment]
Playbook has been run on inventory-next hosts. `datagov_in_service` was set false so no smoke test with service_url. Using empty ckan/datastore DBs, HTTP response looks good to curl request. Still waiting for BSP tickets to be resolved to unblock this work. 

🚧  SSL certificate: RITM0782462 still pending.
🚧  RDS renaming: RITM0783916 created to rename RDS endpoints.

**Comment by FuhuXia** [comment]
🚧  SSL certificate: RITM0782462 still pending. FCS is checking with GSA SecOps team.

**Comment by FuhuXia** [comment]
✅  SSL certificate: RITM0782462 is resolved. 
~~🚧  New ticket RITM0790776 for NetScaler change is created to add new entry **inventory-next.data.gov**.~~ already configured.

**Comment by FuhuXia** [comment]
Playbook run. inventory-next passed smoke test.
